ui/webui/src/app.jsx

```diff
@@ -46,6 +53,46 @@ export const Application = () => {
         return null;
     }
 
+    const wrapWithContext = children => {
+        return (
+            <AddressContext.Provider value={address}>
+                {children}
+            </AddressContext.Provider>
+        );
+    };
+
+    const steps = [
+        {
+            name: _("Installation language"),
+            component: wrapWithContext(<InstallationLanguage />),
```

Comment:
We need to apply the language selection of the user on the onNext. This was previously done in the handleOnContinue handler.